reladomographql/src/test/java/com/gs/reladomo/graphql/GraphqlTestBase.java

```diff
@@ -0,0 +1,19 @@
+package com.gs.reladomo.graphql;
+
+import com.gs.fw.common.mithra.test.ConnectionManagerForTests;
+import com.gs.fw.common.mithra.test.MithraTestResource;
+
+public class GraphqlTestBase {
+
+    public MithraTestResource initReladomo(String configFilename, String dataFile) {

```

Comment:
what happened to the brace `{` formatting in the test classes?

Reply:
We are in the process of switching to the standard github/oracle/google Java code formating.
Hence some code was formated based on the updated code style.